### PR TITLE
Docs improvement: configurations and more

### PR DESCRIPTION
**Pull request by ultmaster**

### CODE REVIEW

File: docs/en_US/Tutorial/NniOnWindows.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # NNI on Windows (experimental feature)
 
-Currently we support local, remote and pai mode on Windows. Windows 10.1809 is well tested and recommended.
+Running NNI on Windows is an experimental feature. Windows 10.1809 is well tested and recommended.
```
Comment (SparkSnail) [inline docs/en_US/Tutorial/NniOnWindows.md]: Is NNI on Windows still experimental?

Comment (xuehui1991) [reply docs/en_US/Tutorial/NniOnWindows.md]: @SparkSnail I think our windows version only contains part of the function of nni?

Comment (SparkSnail) [reply docs/en_US/Tutorial/NniOnWindows.md]: > @SparkSnail I think our windows version only contains part of the function of nni?

Yes, NNI only support `local/remote/pai` on windows, I don't know if we will support all of platform in the feature. In this point, maybe it's reasonable that we call it an experimental feature.

File: docs/en_US/TrainingService/RemoteMachineMode.md [hunk 2]
```diff
@@ -57,24 +57,15 @@ machineList:
...
 
-```bash
-nnictl create --config ~/nni/examples/trials/mnist-annotation/config_remote.yml
-```
-
-to start the experiment.
-
-#### Windows
-Simply filling the `machineList` section and then run:
+Files in `codeDir` will be automatically uploaded to the remote machine. You can run NNI on different operating systems (Windows, Linux, MacOS) to spawn experiments on the remote machines:
```
Comment (QuanluZhang) [inline docs/en_US/TrainingService/RemoteMachineMode.md]: better to mention that only linux is allowed on remote machines

File: docs/en_US/TrialExample/SquadEvolutionExamples.md [hunk 3]
```diff
@@ -112,10 +112,6 @@ trial:
   memoryMB: 32869
   #The docker image to run nni job on OpenPAI
   image: msranni/nni:latest
-  #The hdfs directory to store data on OpenPAI, format 'hdfs://host:port/directory'
-  dataDir: hdfs://10.10.10.10:9000/username/nni
-  #The hdfs directory to store output data generated by nni, format 'hdfs://host:port/directory'
-  outputDir: hdfs://10.10.10.10:9000/username/nni
```
Comment (QuanluZhang) [inline docs/en_US/TrialExample/SquadEvolutionExamples.md]: no need to specify dataDir?

File: docs/en_US/Tutorial/NniOnWindows.md (hunk 1 above)
Comment (QuanluZhang) [reply docs/en_US/Tutorial/NniOnWindows.md]: could remove this sentence. could be replaced with `We have supported NNI on Windows`.

File: docs/en_US/TrialExample/SquadEvolutionExamples.md (hunk 3 above)
Comment (SparkSnail) [reply docs/en_US/TrialExample/SquadEvolutionExamples.md]: yes, the configuration is removed.

File: docs/en_US/Tutorial/NniOnWindows.md (hunk 1 above)
Comment (xuehui1991) [reply docs/en_US/Tutorial/NniOnWindows.md]: But we did not support all functions in Windows.  We should point out which functions we support, which not.

File: docs/en_US/Tutorial/ExperimentConfig.md [hunk 4]
```diff
@@ -130,442 +199,483 @@ machineList:
...
 
-* __trainingServicePlatform__
-  * Description
+* __pai__  submit trial jobs to [OpenPAI](https://github.com/Microsoft/pai) of Microsoft. For more details of pai configuration, please refer to [Guide to PAI Mode](../TrainingService/PaiMode.md)
 
-    __trainingServicePlatform__ specifies the platform to run the experiment, including {__local__, __remote__, __pai__, __kubeflow__}.
+* __kubeflow__ submit trial jobs to [kubeflow](https://www.kubeflow.org/docs/about/kubeflow/), NNI support kubeflow based on normal kubernetes and [azure kubernetes](https://azure.microsoft.com/en-us/services/kubernetes-service/). For detail please refer to [Kubeflow Docs](../TrainingService/KubeflowMode.md)
 
-    * __local__ run an experiment on local ubuntu machine.
+* TODO: explain frameworkcontroller.
```
Comment (QuanluZhang) [inline docs/en_US/Tutorial/ExperimentConfig.md]: frameworkcontroller submit trial jobs to kubernetes. It is built to orchestrate all kinds of applications (e.g., Tensorflow models, PyTorch models, Python scripts) on Kubernetes by a single controller. For more details of configuration for frameworkcontroller mode, please refer to [FrameworkController Docs]()

**Pull request merged**